apps/test-examples/src/Test2380.tsx

```diff
@@ -0,0 +1,19 @@
+import React from 'react';
+import {SvgCss} from 'react-native-svg/css';
+
+const xml = `
+  <svg width="32" height="32" id="mySVG">
+    <style>
+      #mySVG {
+        --my-color: #ff0000;
+        --my-width:150px;
+        --my-height:200px;
+      }
+    </style>
+    <rect fill="var(--my-color)" x="0" y="0" width="var(--my-width)" height="var(--my-height)" />
```

Comment:
I believe `width` and `height` attributes can not be CSS variables. (or at least they don't work in chrome)

Reply:
Yes, you're right. As I see it, it should support: fill, stroke, and color properties.